codec-http/src/test/java/io/netty/handler/codec/http/HttpRequestEncoderTest.java

```diff
@@ -48,6 +48,16 @@ public void testUriWithoutPath2() throws Exception {
     }
 
     @Test
+    public void testUriWithoutPath3() throws Exception {
+        HttpRequestEncoder encoder = new HttpRequestEncoder();
+        ByteBuf buffer = Unpooled.buffer(64);
+        encoder.encodeInitialLine(buffer, new DefaultHttpRequest(HttpVersion.HTTP_1_1, HttpMethod.GET,
+                "http://localhost:9999/?p1=v1"));
```

Comment:
I mean for the test, the name does not match what's being tested (the URI has a path but the name of the test says that it's without one).